.azure-pipelines/all.yml

```diff
@@ -75,6 +75,9 @@ jobs:
     - checkName: nvml
       displayName: nvml
       os: linux
+    - checkName: opa
```

Comment:
`opa` may also refer to other technologies (such as `Oracle Policy Automation`), maybe we should rename it. `open_policy_agent` ?

Reply:
Sounds good to me. I will rename it.